### DISCUSSION TITLE
Pair sonos sub gen3 with first generation play5

### DISCUSSION THREAD

**Discussion by RetroSven** [avantrec/soco-cli#60] [Q&A]
As you probably already know, Sonos has imposed restrictions over which hardware will work with which apps (S1 versus S2).  The Sonos Sub gen3, for example, will only work with the S2 app.  The gen1 and gen2 versions will work with either S1 or S2.  I cannot use the S2 app because I have older Sonos speakers that do not work with it (first gen play5)

My pointed question is - can I buy the gen3 sub, initialize it with the S2 app, and then use soco-cli to manually join it to a stereo pair of gen1 play5's?

**Comment by pwt** [comment]
Unfortunately, no. Paired (or grouped) speakers all need to be part of the same Sonos system. I don't think there's going to be a way of setting up what you're hoping for.
